canonical/ubuntu-advantage-client
# package: depend on ca-certificates

The main functionality in the package requires `ca-certificates` to be
there to make the communication to the Contracts Server possible.

## Test Steps
Build the package, install the new version (for any release), and check it is there via
`apt depends ubuntu-advantage-tools` (:

## Desired commit type

## Checklist
